src/components/Commands.vue

```diff
@@ -636,6 +692,90 @@ export default Vue.extend({
+                let createPermissionSucceeded = false;
+                let alertErrorDetail = "";
+                gdVueComponent.shareGoogleDriveFile()
+                    .then((succeeded) => createPermissionSucceeded = succeeded)
+                    .catch((error) => alertMessage = error)
+                    .finally(() => {
+                        if(createPermissionSucceeded){
+                            // We have set the file public on Google Drive, now we retrieve the share link.
+                            gapi.client.request({
+                                path: "https://www.googleapis.com/drive/v3/files/" + gdVueComponent.saveFileId,
+                                method: "GET",
+                                params: {fields: "webViewLink"},
+                            })
+                                .then((resp) => {
+                                    if(resp.status == 200){
+                                        shareLink = `${window.location}?${sharedStrypeProjectIdKey}=${encodeURIComponent(JSON.parse(resp.body)["webViewLink"])}`;
+                                        alertMessage = this.$i18n.t("appMessage.sharedProjectLinkCopied") as string;
+                                    }
+                                    else{
+                                    // Something happened when we tried to get the public URL of the Google Drive file.
+                                    // We still share the file within Google Drive
+                                        shareLink = `${window.location}?${sharedStrypeProjectTargetKey}=${this.appStore.syncTarget}&${sharedStrypeProjectIdKey}=${this.appStore.currentGoogleDriveSaveFileId}`;
+                                        alertMessage = this.$i18n.t("appMessage.gdPublicShareFailed", (this.$i18n.t("errorMessage.webViewLinkNotRetrieved", resp.status?.toString()??"unknown") as string)) as string;
+                                    }
+                                })
+                                .catch((error) => {
+                                    // Something happened when we tried to get the public URL of the Google Drive file.
+                                    // We still share the file within Google Drive
+                                    shareLink = `${window.location}?${sharedStrypeProjectTargetKey}=${this.appStore.syncTarget}&${sharedStrypeProjectIdKey}=${this.appStore.currentGoogleDriveSaveFileId}`;
+                                    alertMessage = this.$i18n.t("appMessage.gdPublicShareFailed", (this.$i18n.t("errorMessage.webViewLinkNotRetrieved", error) as string)) as string;
+            
+                                })
+                                .finally(() => this.finaliseShareProjectLink(shareLink, alertMessage));                           
```

Comment:
Does this work?  My understanding is that we need to be responding to a user gesture to read or write to the clipboard, but by then we are not responding to a gesture so it won't be able to access the clipboard.